<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCSDKExampleClass</class>
 <widget class="QMainWindow" name="QtCSDKExampleClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>988</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSDKExample</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="tex_username">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="tex_passward">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tex_server">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="tex_conlog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>780</y>
      <width>1151</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>加入房间</string>
    </property>
    <widget class="QPushButton" name="btn_joinRoom">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>加入房间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>房间名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_leave_room">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>退出房间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_getusers">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>获取房间用户</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_users_list">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_room_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QFrame" name="localframe">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="remoteframe">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="remoteframe_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="remoteframe_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>260</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="remoteframe_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="remoteframe_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="remoteframe_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>520</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="remoteframe_7">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>520</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableVideo_cb_local">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>video</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableAudio_cb_local">
       <property name="text">
        <string>audio</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>透明通道</string>
    </property>
    <widget class="QPushButton" name="btn_transbuffer">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>指定用户或服务器广播</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_transbufferroom">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>指定房间广播</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="remoteuser_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>500</y>
      <width>189</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="remote_audio_level_1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="remote_audio_level_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="remote_audio_level_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>470</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="remote_audio_level_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>730</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="remote_audio_level_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>730</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="remote_audio_level_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>730</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="remote_audio_level_7">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>730</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="local_audio_level">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>屏幕共享</string>
    </property>
    <widget class="QPushButton" name="btn_desktopshare">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>全屏幕共享</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_windowshare">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>窗体共享</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>窗体名</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_windowname">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>屏幕号</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_desktopid">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="remote_label_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableVideo_cb_remote_1">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>video</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableAudio_cb_remote_1">
       <property name="text">
        <string>audio</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="remote_label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableVideo_cb_remote_2">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>video</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableAudio_cb_remote_2">
       <property name="text">
        <string>audio</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>490</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="remote_label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableVideo_cb_remote_3">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>video</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableAudio_cb_remote_3">
       <property name="text">
        <string>audio</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>750</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="remote_label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableVideo_cb_remote_4">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>video</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableAudio_cb_remote_4">
       <property name="text">
        <string>audio</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>750</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="remote_label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableVideo_cb_remote_5">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>video</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableAudio_cb_remote_5">
       <property name="text">
        <string>audio</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_13">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>750</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="remote_label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableVideo_cb_remote_6">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>video</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableAudio_cb_remote_6">
       <property name="text">
        <string>audio</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_14">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>750</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="remote_label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableVideo_cb_remote_7">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>video</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableAudio_cb_remote_7">
       <property name="text">
        <string>audio</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>UserLogin()</slot>
  <slot>UserLogout()</slot>
  <slot>JoinRoom()</slot>
 </slots>
</ui>
